packages/platform/platform-multer/src/adapters/fastify/MulterModule.ts

```
@@ -0,0 +1,82 @@
+import {promisify} from "node:util";
+
+import {configuration, constant, injectable} from "@tsed/di";
+import type {FastifyInstance, FastifyReply, FastifyRequest} from "fastify";
+import fp, {type PluginMetadata} from "fastify-plugin";
+import type {IncomingMessage} from "http";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)